### PR TITLE
ios13 update

### PR DESCRIPTION
**Pull request by ps2**
Also includes update to latest dash sdk.

### CODE REVIEW

**Review by nhamming — APPROVED** [review]
LGTM

File: DashKitUI/Views/HUDAssets.xcassets/pod_life/pod_life.imageset/Contents.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "filename" : "Pod Life.pdf"
```
Comment (darinkrauss) [inline DashKitUI/Views/HUDAssets.xcassets/pod_life/pod_life.imageset/Contents.json]: Might want to consider having the file name the same as the asset name.

File: DashKit/DashPumpManager.swift [hunk 2]
```diff
@@ -668,43 +702,44 @@ public class DashPumpManager: PumpManager {
...
-//                    })
-//                }
-//            }
-//        }
+        if mustProvideBLEHeartbeat && !isPeriodicStatusCheckConfigured {
+            self.log.debug("podCommManager periodic status: configuring")
+            podCommManager.configPeriodicStatusCheck(interval: .minutes(1)) { (result) in
+                switch result {
+                case .failure(let error):
+                    self.log.error("podCommManager periodic status check error: %{public}@", String(describing: error))
```
Comment (darinkrauss) [inline DashKit/DashPumpManager.swift:736]: If this happens (and other sources of BLE heartbeat) fail then Loop won't loop. Should we check and warn about this condition? Perhaps at app level if there are no working BLE heartbeat devices? Or is the chance of that so remote?

File: DashKit/DashPumpManager.swift [hunk 3]
```diff
@@ -668,43 +702,44 @@ public class DashPumpManager: PumpManager {
...
-                        self.finalizeAndStoreDoses()
-                        completion(.success(dose))
-                    case .failure(let error):
-                        self.mutateState({ (state) in
-                            state.activeTransition = nil
-                        })
-                        self.finalizeAndStoreDoses()
-                        completion(.failure(DashPumpManagerError(error)))
+                // SDK not allowing us to make calls from a callback thread, so dispatch.
+                DispatchQueue.global(qos: .userInteractive).async {
```
Comment (darinkrauss) [inline DashKit/DashPumpManager.swift:706]: Not main?

File: DashKit/DashPumpManager.swift [hunk 4]
```diff
@@ -668,43 +702,44 @@ public class DashPumpManager: PumpManager {
...
+                        switch result {
+                        case .success(let podStatus):
+                            self.mutateState({ (state) in
+                                state.unfinalizedTempBasal = UnfinalizedDose(tempBasalRate: enactRate, startTime: startDate, duration: duration, scheduledCertainty: .certain)
+                                state.updateFromPodStatus(status: podStatus)
+                                state.activeTransition = nil
+                            })
+                            self.finalizeAndStoreDoses()
+                            completion(.success(dose))
+                        case .failure(let error):
```
Comment (darinkrauss) [inline DashKit/DashPumpManager.swift]: [nit] `.failure` case first in switch?

**Review by darinkrauss — APPROVED** [review]
A few questions to consider, but otherwise LGTM!

File: DashKit/DashPumpManager.swift (hunk 2 above)
Comment (ps2) [reply DashKit/DashPumpManager.swift:736]: A couple things: this configuration (pump driving the heartbeat) is primarily used for testing; the CGM should generally drive things.  We *do* enable it if we don't hear from the CGM in a while, but by that point we can't loop as we do not have fresh cgm data.  Secondly, we do attempt to assert that this is enabled (if needed) several times, like on *any* cgm update, including errors, or when the app is made active.

File: DashKit/DashPumpManager.swift (hunk 3 above)
Comment (ps2) [reply DashKit/DashPumpManager.swift:706]: Yeah, doesn't need to be main.  Could arguably be even lower priority, but there *are* times when temp basal is initiated directly from user.

**Pull request merged**